src/game/MatrixGeneration.ts

```diff
@@ -0,0 +1,249 @@
+import GameConfiguration from "./GameConfiguration";
+
+import seedrandom from "seedrandom";
+
+//TODO should this defined here?
+//TODO how to choose these numbers?
+//TODO is this correct way to provide constants? (what are the naming conventions?)
+const MinMatrixSize = 3;
+const MaxMatrixSize = 8;
+
+const MinNumberSequences = 2;
+const MaxNumberSequences = 7;
+const MinSequenceLength = 2;
+
+const MinBufferSize = 4;
+const MaxBufferSize = 8;
+
+const StepsUntilMatrixSizeIncreased = 10;
+const StepsUntilSequenceNumberIncreased = 3;
+
+const DefaultTimeout = 60_000;
+
+//TODO see how many different values are there
+//TODO see how the possible values are defined
+const numberDifferentMatrixValues = 5;
+const matrixValues = ["A0", "E9", "4C", "8B", "6F"];
+const EmptyMatrixValue = "  ";
```

Comment:
We can probably leave this an empty string and let the UI decide how to display it.

Reply:
Well, this value is currently only used for the sequence generation. During the creation of these, I first create a copy of the previously generated matrix and set the values which are already chosen to this value. I use this to ensure a sequence does not contain the exact same matrix field twice. So for now, this doesn't have any effect on the UI.
However, I could just use "" instead (and probably remove this constant completely).